### PR TITLE
SNOW add custom resolution code parameter

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8773

## Description
Added a new parameter to enable mirroring the resolution code from SNOW to the close reason of XSOAR.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by omerKarkKatz — APPROVED** [review]
Looks good. 
If the user will configure both server_custom_close_code and server_close_custom_state. 
We will use the close code. 
Should we allow configuring both or throw an exception?

File: Packs/ServiceNow/Integrations/ServiceNowv2/README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ If MFA is enabled for your user, follow the next steps:
...
     | Mirrored XSOAR Ticket custom close state code | Define how to close the mirrored tickets in Cortex XSOAR with a custom state. Enter here a comma-separated list of custom closure state codes and their labels (acceptable format example: “10=Design,11=Development,12=Testing”) to override the default closure method. Note that a matching user-defined list of custom close reasons must be configured as a "Server configuration" in Cortex XSOAR. Not following this format will result in closing the incident with a default close reason. | False |
+    | Mirrored XSOAR Ticket custom close resolution code | Define how to close the mirrored tickets in Cortex XSOAR with a custom resolution code. Enter here a comma-separated list of custom resolution codes and their labels (acceptable format example: “10=Design,11=Development,12=Testing”) to override the default closure method. Note that a matching user-defined list of custom close reasons must be configured as a "Server configuration" in Cortex XSOAR. Not following this format will result in closing the incident with a default close reason. | False |
```
Comment (ShirleyDenkberg) [inline Packs/ServiceNow/Integrations/ServiceNowv2/README.md:110]:
```suggestion
    | Mirrored XSOAR Ticket custom close resolution code | Define how to close the mirrored tickets in Cortex XSOAR with a custom resolution code. Enter a comma-separated list of custom resolution codes and their labels (acceptable format example: “10=Design,11=Development,12=Testing”) to override the default closure method. Note that a matching user-defined list of custom close reasons must be configured as a "Server configuration" in Cortex XSOAR. Not following this format will result in closing the incident with a default close reason. | False |
```

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml [hunk 2]
```diff
@@ -185,6 +185,11 @@ configuration:
   name: server_close_custom_state
   type: 0
   required: false
+- additionalinfo: 'Define how to close the mirrored tickets in Cortex XSOAR with a custom resolution code. Enter here a comma-separated list of custom resolution codes and their labels (acceptable format example: “10=Design,11=Development,12=Testing”) to override the default closure method. Note that a matching user-defined list of custom close reasons must be configured as a "Server configuration" in Cortex XSOAR. Not following this format will result in closing the incident with a default close reason.'
```
Comment (ShirleyDenkberg) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml:188]:
```suggestion
- additionalinfo: 'Define how to close the mirrored tickets in Cortex XSOAR with a custom resolution code. Enter a comma-separated list of custom resolution codes and their labels (acceptable format example: “10=Design,11=Development,12=Testing”) to override the default closure method. Note that a matching user-defined list of custom close reasons must be configured as a "Server configuration" in Cortex XSOAR. Not following this format will result in closing the incident with a default close reason.'
```

File: Packs/ServiceNow/ReleaseNotes/2_5_45.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### ServiceNow v2
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+- Added the *Mirrored XSOAR Ticket custom close resolution code* parameter, which allows you to define the custom ServiceNow resolution codes that correspond to the XSOAR incident custom close reasons defined in the server configuration.
```
Comment (ShirleyDenkberg) [inline Packs/ServiceNow/ReleaseNotes/2_5_45.md:6]:
```suggestion
- Added the *Mirrored XSOAR Ticket custom close resolution code* parameter, which allows you to define the custom ServiceNow resolution codes that correspond to the Cortex XSOAR incident custom close reasons defined in the server configuration.
```

**Comment by ShirleyDenkberg** [comment]
@RotemAmit @omerKarkKatz Doc review completed.

**Pull request merged**